packages/@webex/internal-plugin-metrics/test/unit/spec/business/business.ts

```diff
@@ -0,0 +1,120 @@
+import sinon from 'sinon';
```

Comment:
Can we rename this file to be consistent with what we already have?
`business-metrics.ts` instead of `business.ts`

Same comment for `operational.ts` as well

Reply:
Oh yes, good point. Will do, thanks.